src/common/Header/index.js

```diff
@@ -0,0 +1,60 @@
+// @flow
+
+import * as React from 'react';
+import { StyleSheet } from 'react-native';
+import { Header as RNEHeader } from 'react-native-elements';
+import HeaderDrawerButton from './HeaderDrawerButton';
+import HeaderBackButton from './HeaderBackButton';
+import { COLORS, TYPOGRAPHY } from '../../theme';
+
+const styles = StyleSheet.create({
+  innerContainer: {
+    alignItems: 'center',
+  },
+  outerContainer: {
+    backgroundColor: COLORS.primary2,
+    elevation: 2,
+    height: 60,
+    paddingVertical: 0,
+  },
+  title: {
+    ...TYPOGRAPHY.heading,
+    color: COLORS.textInverse,
+  },
+});
+
+type Props = {
+  leftComponent?: React.ComponentType<any> | React.Node,
```

Comment:
So you can do `SomeFooComponent` or `<SomeFooComponent />`? It seems like a weird API.

Reply:
Ah, yeah..I thought RNEHEader support it, it actually should be `React.Element |  some custom type 
 of object  like { icon: {}, style: } or { text: }`..so let it be just React.Element for us.